torch/_inductor/ir.py

```diff
@@ -921,56 +968,35 @@ def fn(index):
                 ranges,
             )
 
-        split_reduction = (
-            is_triton(device)
-            and reduction_type
-            not in {
-                "argmax",
-                "argmin",
-            }
-            and config.split_reductions
+        # triton doesn't support reduce to single element well, so break it up
```

Comment:
nit. This comment is repeated a few lines below, and now it's sort of stale, better remove it.